sdk/alertsmanagement/Azure.ResourceManager.AlertsManagement/CHANGELOG.md

```diff
@@ -1,15 +1,12 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.1 (2023-02-20)
```

Comment:
I believe this PR should contains the property bag [change](https://github.com/Azure/azure-sdk-for-net/blob/45f5c66c6522f971e5acc8f505ca111fb1f4c610/sdk/alertsmanagement/Azure.ResourceManager.AlertsManagement/src/Generated/Models/ServiceAlertCollectionGetAllOptions.cs#L11).

Please add `- Introduced property bag for the methods with more than 5 parameters.` in Features Added.

Reply:
Added